compiler/src/iree/compiler/Codegen/LLVMCPU/test/convert_to_llvm.mlir

```diff
@@ -102,13 +102,7 @@ module attributes {
 }
 //      CHECK: llvm.func @paramstruct_cconv_with_extra_fields_and_executable_target(!llvm.ptr)
 //      CHECK: llvm.func @bar
-//  CHECK-DAG:   %[[F16C:.+]] = llvm.mlir.constant(16384 : i64) : i64
```

Comment:
unfortunately makes the test more opaque (vs "fma was set so the fma bit should be OR'ed in") but the savings in readability of IR feels worth it!